be/src/runtime/data_stream_recvr.cc

```diff
@@ -573,8 +572,7 @@ bool DataStreamRecvr::is_data_ready() {
 }
 
 Status DataStreamRecvr::add_chunks(const PTransmitChunkParams& request, ::google::protobuf::Closure** done) {
-    MemTracker* prev_tracker = tls_thread_status.set_mem_tracker(_instance_mem_tracker.get());
```

Comment:
Can it be replaced with bthread memtracker?

Reply:
Yes